CMakeLists.txt

```
@@ -15,7 +15,7 @@
 ###############################################################################
 # CMake build rules for FastCDR                                               #
 ###############################################################################
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.15)
```

If the bug was resolved in 3.15.5, shouldn't we set that version as minimum required?
```suggestion
cmake_minimum_required(VERSION 3.15.5)
```

---

It is a bug that produces only a warning. I don't think we have to put the PATCH version. We are telling the people we tested against the CMake 3.15 API. Usually if someone uses a product minor version, he will use the last patched version. That decision doesn't concern us. For example, our CI uses the last patched version 3.15.7.